packages/cnpj-dv/tests/Specs/CnpjCheckDigits.spec.php

```
@@ -206,82 +206,100 @@
                 $invalid = null;
 
-                expect(fn () => new CnpjCheckDigits($invalid))->toThrow(CnpjCheckDigitsInputTypeError::class);
+                expect(function () use ($invalid) {
+                    new CnpjCheckDigits($invalid);
+                })->toThrow(CnpjCheckDigitsInputTypeError::class);
             });
 
-            it('throws CnpjCheckDigitsInputTypeError for object input', function () {
+            it('throws `CnpjCheckDigitsInputTypeError` for object input', function () {
                 /** @var mixed $invalid */
                 $invalid = (object) ['cnpj' => '12345678901'];
```

PHPStan (2.1.54)

[warning] PHPDoc tag `@var` with type mixed is not subtype of native type object{cnpj: '12345678901'}&stdClass.


(varTag.nativeType)